### ISSUE TITLE
Cannot setup cluster in InfluxDB 0.13

### ISSUE THREAD

**Issue by mohan111raj** [influxdata/influxdb#6842]
Hi all,

I am trying to create a cluster with 3 nodes. I tried all steps mentioned in the https://docs.influxdata.com/influxdb/v0.9/clustering/cluster_setup/ and https://docs.influxdata.com/influxdb/v0.11/clustering/cluster_setup/. 

I used the below command to start the server: run: flag provided but not defined: -join
**sudo influxd -config /etc/influxdb/influxdb.conf -join Ip1:8091,Ip2:8091,Ip3:8091**

but then I got the error : **run: flag provided but not defined: -join**
But after completing everything I got error "Failed to connect to localhost:8068" in starting the servers second and third nodes.

So I changed the bind-address ="localhost:8086" in /etc/influxdb/influxdb.conf under "http" parameters.
after this I was able to start the server. I have given the "SHOW SERVERS" and I got the following error in all 3 three nodes.

> SHOW SERVERS
ERR: error parsing query: found SERVERS, expected CONTINUOUS, DATABASES, DIAGNOSTICS, FIELD, GRANTS, MEASUREMENTS, QUERIES, RETENTION, SERIES, SHARD, SHARDS, STATS, SUBSCRIPTIONS, TAG, USERS at line 1, char 6

Please help me to resolve this issue. 
If I am doing anything wrong in setting up the cluster **please give me a clue to setup a cluster.**

Thanks in advance.

**Comment by toddboom** [comment]
Clustering is no longer available in the open-source version of InfluxDB. Please see our announcement from March 2016 about this change:
https://influxdata.com/blog/update-on-influxdb-clustering-high-availability-and-monetization/

To achieve HA with our open-source tools, check out InfluxDB Relay:
https://github.com/influxdata/influxdb-relay

**Comment by mohan111raj** [comment]
Hi toodboom,

Thanks for the answer. It saved lot of time.

For now, I am testing the clustering availability and performance over mongodb. If the clustering availability meets our requirement we'll be moving from mongo to influxdb.

I'll try with influx 0.11 for now. Just wondering how to install v0.11? I tried installing it in the past in ubuntu and it installs the latest 0.13 by default when I used installation instruction from influx docs.

Thanks.
